ls1intum/Artemis
# Show non-released exercises on team page for tutors and above

### Checklist

### Description
Right now, the team page only shows exercises that are visible to students, even for tutors and instructors. They should be able to see all exercises though, including non-released ones.

### Steps for Testing
1. Log in to Artemis as a tutor / instructor
2. Navigate to Course Management
3. Click on Exercises for a Course that has a Team Exercise which is not released yet
4. Click on Teams for that Team Exercise
5. Open the Team page of one of the teams
6. Check that the non-released exercise is shown in the participations table
